src/components/FolioNumberInput/FolioNumberInput.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <v-form id="folio-number-form" class="ml-8" ref="folioForm" v-model="folioFormValid">
```

Comment:
You probably should remove the margin here and let the parent take care of it, which the parent can do like this: `<FolioNumberInput class="ml-8">...`

Reply:
Yes, good catch!